usr/share/rear/restore/NETFS/default/51_set_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# restore capabilities if capabilities are in the backup
+	if test -s $VAR_DIR/recovery/capabilities ; then
+		Log "Restoring Capabilities."
+		if which setcap >/dev/null 2>&1 ; then
+			while read file cap ; do
+				setcap $cap ${TARGET_FS_ROOT}/${file}	
```

Comment:
Please quote the file name so that we handle filenames with blanks in them.  In fact, I think that `getcap` and `setcap` will lead to severe problems with bad filenames. Consider this example:

```
$ getcap -r /tmp
/tmp/foo "= foo = cap_net_raw+ep

```

IMHO it is nearly impossible to parse this output in a sane way. I know that this won't affect the system files that you wrote this code for. Maybe we should somehow detect such cases and warn the user that we can't handle it.